modules/acm/src/main/java/edu/gemini/epics/acm/CaTaskControlImpl.java

```diff
@@ -21,7 +21,9 @@
 import java.util.concurrent.ScheduledThreadPoolExecutor;
 import java.util.concurrent.TimeUnit;
 
-public class CaTaskControlImpl implements CaTaskControl {
+interface TaskControlWithResoource extends CaTaskControl, CaResource {}
```

Comment:
`Resoource`?

Reply:
`CaResource` is just an interface for objects that need to be "unbinded" at the end of their lifetime.